README.md

```diff
@@ -86,14 +86,20 @@ Initialize the hub by deploying the hub side resources to manage clusters.
 
 it returns the command line to launch on the spoke to join the hub.
 
+> NOTE: Do not run init command against a multicluster-controlplane instance. It is already a initialized hub onstart. Instead, Using `clusteradm get token --use-bootstrap-token` to get the join command
+
 ### join
 
 Install the agent on the spoke.
 
-`clusteradm join --hub-token <token> --hub-apiserver <hub_apiserver_url> --cluster-name c1`
+`clusteradm join --hub-token <token> --hub-apiserver <hub_apiserver_url> --cluster-name c1 [--ca-file <path-to-ca-file>] [--force-internal-endpoint-lookup]`
 
 it returns the command line to launch on the hub the accept the spoke onboarding.
 
+> NOTE: While clusteradm could fetch any ca data from given `--hub-apiserver`, then `--ca-file` flag should be set.  
```

Comment:
Not clear what do you mean. Why --ca-file should be used and in which case?

Reply:
done